README.md

```diff
@@ -44,24 +44,38 @@ Fly is extremely extensible, so _anything_ can be a task. Our core system will a
 <h2>Table of Contents</h2>
 <details>
 <summary>Table of Contents</summary>
-- [Features](#features)
```

Comment:
sorry, what do you mean with `Can you just add new line here`?
just write each link (markown or tag) in a new line without a list?